Changes.md

```diff
@@ -33,6 +33,7 @@ API
 - PathColumn : Added support for a different icon to be shown when a cell is hovered.
 - SetUI : Removed `ui:scene:acceptsSet*` metadata from entries that are promoted along with a plug.
 - MetadataAlgo : Added `isPromotable()` method.
+- TweakPlug : Support list operations for `PathMatcherData`
```

Comment:
We don't use the imperative mood (or whatever it's called) in Changes.md, and we're missing a period. Maybe this?

- TweakPlug : Added PathMatcherData compatibility in `Replace`, `Create`, `ListAppend` and `ListPrepend` modes.

We also need to mention the PlugAlgo improvement in the API section.